### PR TITLE
Correcting some grammatical mistakes in the design docs

### PR DESCRIPTION
**Pull request by abhinavarora**

### CODE REVIEW

**Comment by NorthStar** [comment]
I'd say Figure 1., Figure 2. for consistency @abhinavarora otherwise all the changes are improvements.

**Comment by abhinavarora** [comment]
MAde changes as per feedback. @NorthStar and @dzhwinter, please review the changes again.

File: paddle/framework/backward.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## Motivation
 
-In Neural Network, many model is solved by the the backpropagation algorithm(known as BP) at present. Technically it caculates the gradient of the loss function, then distributed back through the networks. Follows the chain rule, so we need a module chains the gradient operators/expressions together with to construct the backward pass. Every forward network needs a backward network to construct the full computation graph, the operator/expression's backward pass will be generated respect to forward pass. 
+In Neural Network, most models are solved by the the backpropagation algorithm(known as BP) at present. Technically, it calculates the gradient of the loss function, then propagates it back through the networks following the chain rule. Hence we need a module that chains the gradient operators/expressions together to construct the backward pass. Every forward network needs a backward network to construct the full computation graph. The operator/expression's backward pass will be generated with respect to the forward pass.
```
Comment (NorthStar) [inline paddle/framework/backward.md]: At the moment, most neural network models are solved by the the backpropagation algorithm (known as **BP**). Technically, BP calculates the gradient of the loss function and propagates the gradient back through the network, following the chain rule. Hence we need a module that chains the gradient operators or expressions together to construct the backward pass. Every forward network needs a backward network to construct the full computation graph. The operator or expression's backward pass will be generated with respect to the forward pass.

File: paddle/framework/backward.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ## Motivation
 
-In Neural Network, many model is solved by the the backpropagation algorithm(known as BP) at present. Technically it caculates the gradient of the loss function, then distributed back through the networks. Follows the chain rule, so we need a module chains the gradient operators/expressions together with to construct the backward pass. Every forward network needs a backward network to construct the full computation graph, the operator/expression's backward pass will be generated respect to forward pass. 
+In Neural Network, most models are solved by the the backpropagation algorithm(known as **BP**) at present. Technically, BP calculates the gradient of the loss function, then propagates it back through the networks following the chain rule. Hence we need a module that chains the gradient operators/expressions together to construct the backward pass. Every forward network needs a backward network to construct the full computation graph. The operator/expression's backward pass will be generated with respect to the forward pass.
```
Comment (dzhwinter) [inline paddle/framework/backward.md]: by the the backpropagation => by the backpropagation

**Comment by NorthStar** [comment]
LGTM!

File: paddle/framework/backward.md [hunk 3]
```diff
@@ -48,49 +48,49 @@ The function `BuildGradOp` will sequentially execute following processes:
...
 
-​	pic 2. Replace sharing variable's gradient with `Add` operator.
+​	Figure 2. Replace sharing variable's gradient with `Add` operator.
 
 </p>
 
-​	Because our framework finds variables accord to their names, we need to rename the output links. We add a suffix of number to represent its position in clockwise. 
+​	Because the framework finds variables according to their names, we need to rename the output links. We add an integer suffix to represent its position in the clockwise direction. 
 
 5. Part of Gradient is Zero.
```
Comment (dzhwinter) [inline paddle/framework/backward.md]: Maybe In our implement => In our implementation? I am not sure about the grammar error. :)
